we.go

```diff
@@ -562,33 +568,24 @@ func marshalLang(rv map[string]interface{}, r *resp) error {
 }
 
 func main() {
-	var numdays = 3
-
 	if len(config.APIKey) == 0 {
 		log.Fatal("No API key specified. Setup instructions are in the README.")
 	}
 	params = append(params, "key="+config.APIKey)
 
-	for _, arg := range os.Args[1:] {
-		if v, err := strconv.Atoi(arg); err == nil && len(arg) == 1 {
-			numdays = v
-		} else {
-			config.City = arg
-		}
-	}
```

Comment:
I want to retain the feature to specify an alternate location or number of days without using flags (for brevity). Just replace the loop through os.Args with a loop through flag.Args(). Flags should take precedence over non-flag arguments, so `wego -days 1 -city Paris 4 'New York'` should return one day of forecast for paris. However `wego 4 'New York'` should still return 4 days of forecast for New York.